lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -439,6 +451,25 @@ class PatternStopContents extends Component<Props, State> {
             />
           </Col>
         </Row>
+        <Row>
+          <Col xs={12}>
+            <FormGroup
+              controlId='stopHeadsign'
+              bsSize='small'>
+              <ControlLabel
+                // TODO: internationalize this
+                title={'Headsign that overrides trip headsign between stops.'}
+                className='small'>Stop headsign</ControlLabel>
+              <FormControl
+                type='text'
+                // TODO: internationalize this
+                placeholder={'Outbound'}
+                onChange={this._onHeadsignChange}
+                value={patternStop.stopHeadsign || ''}
```

Comment:
Sort props

Reply:
Sorted in https://github.com/ibi-group/datatools-ui/pull/890/commits/ddf848719158b3500fd6520202ec781d5302dd5d